embedding/csharp/Tizen.Flutter.Embedding/Tizen.Flutter.Embedding.csproj

```diff
@@ -2,18 +2,30 @@
 
   <PropertyGroup>
     <OutputType>library</OutputType>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;tizen40;tizen90</TargetFrameworks>
   </PropertyGroup>
 
   <PropertyGroup>
-    <TizenApiVersion Condition="'$(TizenApiVersion)' == ''">4.0</TizenApiVersion>
+    <PackageId>Tizen.Flutter.Embedding</PackageId>
+    <Version>1.0.0</Version>
+    <Authors>Samsung Electronics</Authors>
+    <Description>Provides APIs for embedding Flutter into Tizen apps.</Description>
+    <PackageLicenseExpression>BSD-3-Clause</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/flutter-tizen</PackageProjectUrl>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
   </PropertyGroup>
 
-  <ItemGroup Condition="$(TizenApiVersion) &lt; 6.5">
+  <ItemGroup>
+    <Content Include="Tizen.Flutter.Embedding.targets" PackagePath="build" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'tizen40'">
     <PackageReference Include="Tizen.NET" Version="4.0.1.14164" />
   </ItemGroup>
 
-  <ItemGroup Condition="$(TizenApiVersion) &gt;= 6.5">
+  <ItemGroup Condition="'$(TargetFramework)' != 'tizen40'">
     <PackageReference Include="Tizen.NET" Version="9.0.0.16760" />
   </ItemGroup>
```

Comment:
I think we don't need to consider the TargetFramework to refer `Tizen.NET` at this time. 
`Tizen.NET` will choose proper `Tizen.NET.APIx` package automatically.

Reply:
For #418, We may need conditions later to include NUI files with followings..  
- TargetFrameworkIdentifier == 'Tizen' and TargetFrameworkVersion > '9.0'
- TargetFrameworkIdentifier == 'NETStandard' or TargetFrameworkIdentifier == 'NETCoreApp'

Reply:
Fixed. Thanks! The following lines will be added in the future.

```xml
  <ItemGroup Condition="'$(TargetFramework)' == 'tizen40'">
    <Compile Remove="NUIFlutterView.cs" />
  </ItemGroup>
```

Reply:
Rebased and added the lines (9f10d2f10261eb95e9465b73e310dc3b9756607e).